client/src/pages/CreateMatch.tsx

```diff
@@ -83,33 +76,6 @@ const CreateMatch: React.FC<createMatchInterface> = ({
     }
   };
 
-  const getStadiumMatchs = async (id: number) => {
-    const matchesFetch = await fetch(`/api/v1/stadiums/matches/${id}`);
-    const stadMatches = await matchesFetch.json();
-    if (stadMatches.status === 401) {
-      setValidateError(stadMatches.data);
-    }
-    let convertedMatches;
-    if (Array.isArray(stadMatches.data)) {
-      convertedMatches = stadMatches?.data?.map((event: prevInterface) => {
-        return {
-          title: event.title,
-          start: event.startDate,
-          end: event.endDate,
-          description: event.description,
-          seats: event.seats,
-        };
-      });
-    }
-
-    setMatches(convertedMatches);
-  };
-  useEffect(() => {
-    if (StadiumId > 0) {
-      getStadiumMatchs(StadiumId);
-    }
-  }, [StadiumId]);
-
   useEffect(() => {
```

Comment:
What is the purpose of `useEffect`, the fetch will work just one time, it won't update